scss/overrides/_forms.scss

```diff
@@ -0,0 +1,14 @@
+.form-control {
+  box-shadow: unset;
+}
+
+.btn-sm {
```

Comment:
Instead of `btn-sm`, can we do it for `btn`?
Also this should go under `_buttons.scss`